libs/vkd3d/command.c

```diff
@@ -1539,8 +1633,15 @@ HRESULT d3d12_shared_fence_create(struct d3d12_device *device,
         return hr;
     }
 
+    pthread_mutex_init(&object->mutex, NULL);
+    list_init(&object->events);
+    object->last_value = 0;
+    object->stop = 0;
+
     d3d12_device_add_ref(object->device = device);
 
+    pthread_create(&object->thread, NULL, vkd3d_shared_fence_worker_main, object);
```

Comment:
I would prefer that a thread is spawned on first waiter that actually requires a thread.